source/LICENSE.md

```diff
@@ -1,3 +1,6 @@
+LICENSE
```

Comment:
Is this LICENSE relevant anymore?

Reply:
I'm not a licensing expert so I was going to leave that one up to you to decide :)

Reply:
We don't use Hyde Jekyll anymore so this license does not apply